api/src/test/java/org/pmiops/workbench/api/AppsControllerTest.java

```diff
@@ -109,6 +110,24 @@ public void testCreateAppFail_validateActiveBilling() {
         ForbiddenException.class, () -> controller.createApp(WORKSPACE_NS, createAppRequest));
   }
 
+  @Test
```

Comment:
Could you add "enableRStudio = true" to this one, to show that it doesn't help, and the equivalent to the next one?

I'd also like to see success cases for each.

Reply:
Done 👍